### ISSUE TITLE
Nested map causes error

### ISSUE THREAD

**Issue by thasmo** [sasstools/sass-lint#531]
Using `gulp-sass-lint` this SCSS ...
```scss
$base-colors: (
	'blue': (
		'base': #00f,
	),
	'red': (
		'base': #f00,
	),
);
```
... causes this error:
```sh
TypeError: Cannot read property 'forEach' of undefined
    at Function.sassLint.warningCount (S:\Project\web\gulp.boilerplate\node_modules\sass-lint\index.js:65:10)
    at Function.sassLint.resultCount (S:\Project\web\gulp.boilerplate\node_modules\sass-lint\index.js:83:23)
    at Function.sassLint.outputResults (S:\Project\web\gulp.boilerplate\node_modules\sass-lint\index.js:220:12)
    at DestroyableTransform._transform (S:\Project\web\gulp.boilerplate\node_modules\gulp-sass-lint\index.js:64:10)
    at DestroyableTransform.Transform._read (S:\Project\web\gulp.boilerplate\node_modules\readable-stream\lib\_stream_transform.js:172:10)
    at DestroyableTransform.Transform._write (S:\Project\web\gulp.boilerplate\node_modules\readable-stream\lib\_stream_transform.js:160:12)
    at doWrite (S:\Project\web\gulp.boilerplate\node_modules\readable-stream\lib\_stream_writable.js:335:12)
    at writeOrBuffer (S:\Project\web\gulp.boilerplate\node_modules\readable-stream\lib\_stream_writable.js:321:5)
    at DestroyableTransform.Writable.write (S:\Project\web\gulp.boilerplate\node_modules\readable-stream\lib\_stream_writable.js:248:11)
    at DestroyableTransform.ondata (S:\Project\web\gulp.boilerplate\node_modules\readable-stream\lib\_stream_readable.js:583:20)
```

**Comment by thasmo** [comment]
This works:
```scss
$base-colors: (
    'blue': #00f,
    'red': #f00,
);
```

**Comment by thasmo** [comment]
Oh, seems like nesting in general creates this error. Same error for this:
````scss
.footer {
	.bottom {
		display: block;
	}
}
```

**Comment by DanPurdy** [comment]
Sorry but there must be something else going on here. I've tested your examples and both work. We have countless examples of nested blocks in our tests too.

What version of sass-lint are you using?

**Comment by thasmo** [comment]
I'm using `gulp-sass-lint` version `1.1.1` which should install `sass-lint` version `1.5.0`, no? That's what it says in `node_modules/sass-lint/package.json`: `"version": "1.5.0"`.

**Comment by DanPurdy** [comment]
Yep. What platform are you working on? Windows? OSX?

hard tabs / soft tabs?

**Comment by thasmo** [comment]
Running on Windows 10 64-bit, Node 5.6.0, using hard tabs.

**Comment by DanPurdy** [comment]
Ok, could you try soft tabs/spaces in that file and see if you get the same error?

I;ve got a feeling it will be down to the hard tabs not working correctly in the current version of our AST. If it is this then #495 will look to fix it.

**Comment by thasmo** [comment]
True, spaces work.

**Comment by DanPurdy** [comment]
Ok, I'll label this as an AST issue for now then. It's a long standing problem that we should see eliminated in our 1.6.0 update which will land once i get round to finishing our last rule update.

**Comment by thasmo** [comment]
Thanks! :)

**Comment by DanPurdy** [comment]
No problem, thanks for the report :+1:

**Comment by bgriffith** [comment]
@thasmo Fixed in 1.6

**Comment by thasmo** [comment]
Yay, awesome!
